specs/agents/metrics-otel.md

```diff
@@ -0,0 +1,175 @@
+ * A bucket based histogram with explicitly provided bucket boundaries
+ * An exponential histogram with predefined, exponentially increasing bucket boundaries
+
+In the SDK, the default aggregation to use for each instrument type (counter, gauge, histogram, etc) is defined by the Exporter / MetricsReader. However, users can explicitly override the aggregation per metric by using [Views](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#view). Note that views are an SDK concept and are not part of the OpenTelemetry API.
+
+Agents MUST use the OpenTelemetry SDK [default aggregations per instrument type](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#default-aggregation) as default aggregations for their exporter:
+
+| **Instrument Type**         | **Aggregation**            |
+|-----------------------------|----------------------------|
+|  Counter                    | Sum                        |
+|  Asynchronous Counter       | Sum                        |
+|  UpDownCounter              | Sum                        |
+|  Asynchronous UpDownCounter | Sum                        |
+|  Asynchronous Gauge         | Last Value                 |
+|  Histogram                  | Explicit Bucket Histogram  |
+
+### Histogram Aggregation
+
+The OpenTelemetry SDK uses the following prometheus-inspired bucket boundaries for histograms by default:
+```
+[0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000]
+```
+
+Because IntakeV2 histogram serialization skips empty buckets, we are free to use histogram boundaries which are more accurate and cover a greater range. Therefore, agents SHOULD use the following histogram boundaries by default instead:
+
+```
+[0.00390625, 0.00552427, 0.0078125, 0.0110485, 0.015625, 0.0220971, 0.03125, 0.0441942, 0.0625, 0.0883883, 0.125, 0.176777, 0.25, 0.353553, 0.5, 0.707107, 1, 1.41421, 2, 2.82843, 4, 5.65685, 8, 11.3137, 16, 22.6274, 32, 45.2548, 64, 90.5097, 128, 181.019, 256, 362.039, 512, 724.077, 1024, 1448.15, 2048, 2896.31, 4096, 5792.62, 8192, 11585.2, 16384, 23170.5, 32768, 46341.0, 65536, 92681.9, 131072]
+```
+
+These boundaries are an exponential scale base `sqrt(2)` rounded to six significant figures.
+
+Agents MUST allow users to configure the default histogram boundaries via the `custom_metrics_histogram_boundaries` configuration option:
+
+|                |                                          |
+|----------------|------------------------------------------|
+| Type           | `double list`                            |
+| Default        | <see base `sqrt(2)` boundaries above>    |
+| Dynamic        | `false`                                  |
```

Comment:
nit: FWIW, the Java implementation has `.dynamic(true)`, but it is defined false here. This doesn't impact the Node.js agent as it doesn't have the concept of "dynamic" config vars separate from "Central config" vars.

Reply:
> FWIW, the Java implementation has .dynamic(true)

Thanks for pointing that our, that's a bug in the Java implementation. The option being dynamic would mean that metrics created before the change of the option would have still the old buckets while newer metrics get the new buckets, which I think is a bad thing.